README.md

```diff
@@ -236,485 +286,1796 @@ const {
 
 request()
   .then((response) => {
-    /* do your own manipulation here */
-    const filteredMessages = doStuffToMessages(response.data.inAppMessages);
+    // Do your own manipulation here 
+    const filteredMessages = yourOwnSortingAndFiltering(response.data.inAppMessages);
 
-    /* also feel free to take advantage of the sorting/filtering methods used internally */
+    // Or, feel free to take advantage of the sorting/filtering methods used 
+    // internally
     const furtherManipulatedMessages = sortInAppMessages(
       filterHiddenInAppMessages(response.data.inAppMessages)
     ) as InAppMessage[];
 
-    /* then display them whenever you want */
+    // Then display them whenever you want
     triggerDisplayMessages(furtherManipulatedMessages);
   })
   .catch();
 ```
 
-:rotating_light: PLEASE NOTE, If you choose the `deferred` option, the SDK will
-**not** do any filtering or sorting on the messages internally. You will get the
-messages exactly as they come down from the API, untouched. This means you may
-(for example) show in-app messages marked `read` or show the messages in the
-wrong order based on `priority`.
+:rotating_light: With the `deferred` option, the SDK does **not** filter or sort
+the messages. The messages come back exactly as retrieved from the API, without
+modification. This means you may, for example, show in- app messages marked as
+`read`, or show the messages in the wrong order (based on `priority`).
```

Comment:
Do you think we can replace "wrong order" here? I know this wording didn't change, but reading it now, it sounds like priority based sorting is considered "wrong"? Perhaps it should say something along the lines of "alternative" or "non-default"? (I don't like that latter suggestion btw, but hopefully it's clear what I'm getting at)

Reply:
Totally agree, will make a change.